src/components/accordion/__tests__/CdrAccordionGroup.spec.js

```diff
@@ -34,6 +34,17 @@ describe('CdrAccordionGroup', () => {
     it('renders correctly', () => {
       expect(wrapper.element).toMatchSnapshot()
     });
+  });
+
+  describe('component unit tests', () => {
```

Comment:
Added another describe here since it makes sense to separate these tests from the snapshot ones